test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntrospectionTest.cs

```diff
@@ -19,5 +19,37 @@ public class BuildIntrospectionTest : MSBuildIntegrationTestBase
             Assert.BuildOutputContainsLine(result, $"UpToDateCheckInput: {Path.Combine("Views", "Home", "Index.cshtml")}");
             Assert.BuildOutputContainsLine(result, $"UpToDateCheckInput: {Path.Combine("Views", "_ViewStart.cshtml")}");
         }
+
+        [Fact]
+        [InitializeTestProject("SimpleMvc")]
+        public async Task RazorSdk_SetsPreserveCompilationContextForProjectsWithCshtmlFiles()
```

Comment:
Do we need these introspection tests now that we have the 'real thing'? IMO they don't add much value and can be removed.

Reply:
Yeah, I can take them out. Faster test runs